src/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ set_property(CACHE LibtorrentRasterbar_DIR PROPERTY TYPE PATH)
 find_package(Boost ${minBoostVersion} REQUIRED)
 find_package(OpenSSL ${minOpenSSLVersion} REQUIRED)
 find_package(ZLIB ${minZlibVersion} REQUIRED)
-find_package(Qt5 ${minQtVersion} REQUIRED COMPONENTS Core Network Xml LinguistTools)
+find_package(Qt5 ${minQtVersion} REQUIRED COMPONENTS Core Network Sql Xml LinguistTools)
```

Comment:
Super minor nitpick and not really related, but would you mind sorting these alphabetically? `Core LinguistTools Network Sql Xml`

Reply:
I have a counter unrelated question, for what purposes do we use `LinguistTools` in the build process? Why is it required here?

Everything else is already arranged alphabetically. `LinguistTools` is the only one that is out of order, so I thought it was intentional.

Reply:
@glassez

It is possible that `LinguistTools` is not required here, not sure, it was "grandfathered in" from the previous build system implementation (but I would say it is probably by something related to translations). You could make a test PR without it to see if qBittorrent builds successfully on all platforms without it. As for the order, it doesn't matter here, `find_package()` does not care. I just think it is more convenient for readability if the arguments are in alphabetical order.

Reply:
>I just think it is more convenient for readability if the arguments are in alphabetical order.

Anyway `LinguistTools` is logically different thing than other components from this list so it looks good for me in existing way.

Reply:
@glassez Alright, but did you end up determining whether `LinguistTools` is really necessary for the build or not? If it is, feel free to mark this as resolved. And thanks for the explanation below.

Reply:
>did you end up determining whether `LinguistTools` is really necessary for the build or not?

It is for `qt5_add_translation`.

Reply:
@FranciscoPombal
Don't forget to approve the PR again.